src/session.ts

```diff
@@ -188,12 +188,12 @@ async function refreshSession({
 
   return {
     sessionId,
-    user: session.user,
```

Comment:
Here we were accidentally returning the user, impersonator and accessToken from the _old_ session instead of the new one.